shared/src/enums/error-messages.enum.ts

```diff
@@ -4,6 +4,9 @@ const ErrorMessages = {
     INCORRECT_EMAIL: 'Incorrect email.',
     EMAIL_ALREADY_EXISTS: 'Email is already taken.',
     PASSWORDS_NOT_MATCH: 'Passwords do not match.',
+    ROUTE_ADMIN_ONLY: 'This route is for administrators only.',
```

Comment:
You don't need this in this PR. This is another task. Please don't go beyond the scope of your task !